static/app/views/insights/types.tsx

```diff
@@ -255,6 +255,13 @@ export enum SpanIndexedField {
   INP = 'measurements.inp',
   INP_SCORE = 'measurements.score.inp',
   INP_SCORE_WEIGHT = 'measurements.score.weight.inp',
+  INP_SCORE_RATIO = 'measurements.score.ratio.inp',
```

Comment:
👀 is this used anywhere?

Reply:
Forgot to remove this. It's currently not used anywhere.
For context, I have plans to refactor the UI to use this measurement instead since it reduces the need to do calculation between multiple measurements. I'll save this change for a future pr.